locustfile.py

```diff
@@ -0,0 +1,225 @@
+"""
+locustfile for simulating API requests
+
+to run, make sure LOCUST_API_KEY is in your env and contains
+a valid API key for the API environment being load tested.
+
+example command:
+$ LOCUST_API_KEY="myapikey" locust --host="https://my-api-env.mbtace.com"
+
+for reference, prod handles ~250 requests/second with 3 instances
+"""
+import os
+from locust import HttpUser, task, between
+from random import randint
+
+
+class ApiUser(HttpUser):
+
+    wait_time = between(0.5, 3)
+
+    # TODO is there a better way to pass this in?
+    api_key = os.environ["LOCUST_API_KEY"]
+    headers = {"x-api-key": api_key}
+
+    # reusable data
+    green_line_routes = ["Green-B", "Green-C", "Green-D", "Green-E"]
+    light_rail_routes = green_line_routes + ["Mattapan"]
+    heavy_rail_routes = ["Red", "Orange", "Blue"]
+    subway_routes = heavy_rail_routes + light_rail_routes
+    key_bus_routes = ["1", "15", "22", "23", "28", "32", "39", "57",
+                      "66", "71", "73", "77", "111", "116", "117"]
+    solari_screen_stations = [
+        # forest hills
+        "10642",
+        # nubian square
+        "place-dudly",
+        # sullivan square upper busway
+        ["29001", "29002", "29003", "29004", "29005", "29006"],
+        # ashmont
+        "334",
+        # wonderland
+        "15795",
+        # ruggles lower busway
+        ["17862", "17863"],
+    ]
+    eink_screen_stations = [
+        # washington square (Green-B)
+        "place-bcnwa",
+        # museum of fine arts (Green-E)
+        "place-mfa",
+    ]
+
+    # helper function to construct JSONAPI queries
+    def api_request(self, endpoint, **kwargs):
+
+        if "name" in kwargs.keys():
+            name = kwargs["name"]
+        else:
+            name = None
```

Comment:
`name = kwargs.get("name")` but really what you want is to have this provided as default keyword arguments.

```python
def api_request(self, endpoint, name=None, filters=None, fields=None, include=None, sort=None):
```